com.github.debauchee.barrier.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id" : "com.github.debauchee.barrier",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "5.12",
+    "sdk" : "org.kde.Sdk",
+    "command" : "barrier",
+    "rename-icon" : "barrier",
+    "rename-desktop-file" : "barrier.desktop",
```

Comment:
Would be nice to rename those upstream

Reply:
Are you saying that our build system should produce an icon with the appropriate name?